contracts/FlowIDTableStaking.cdc

```diff
@@ -1212,16 +1212,14 @@ pub contract FlowIDTableStaking {
                 
                 nodeRecord.tokensRewarded.deposit(from: <-rewardsVault.withdraw(amount: nodeReward))
 
-                emit RewardsPaid(nodeID: rewardBreakdown.nodeID, amount: nodeReward)
-
                 for delegator in rewardBreakdown.delegatorRewards.keys {
                     let delRecord = nodeRecord.borrowDelegatorRecord(delegator)
                     let delegatorReward = rewardBreakdown.delegatorRewards[delegator]!
                         
                     delRecord.tokensRewarded.deposit(from: <-rewardsVault.withdraw(amount: delegatorReward))
-
-                    emit DelegatorRewardsPaid(nodeID: rewardBreakdown.nodeID, delegatorID: delegator, amount: delegatorReward)
                 }
+
+                emit RewardsPaid(nodeID: rewardBreakdown.nodeID, amount: nodeReward, delegatorAmounts: rewardBreakdown.delegatorRewards)
```

Comment:
Have we communicated this proposal to the community at all yet?

Reply:
Not yet, but I wanted a review first, then we'll start sharing it around for feedback and approvals